src/d3d9/d3d9_device.cpp

```diff
@@ -4702,6 +4702,27 @@ namespace dxvk {
     if (unlikely(!desc.IsLockable))
       return D3DERR_INVALIDCALL;
 
+    D3DRESOURCETYPE type = pResource->GetType();
+
+    // Volume textures in D3DPOOL_DEFAULT must have the D3DUSAGE_DYNAMIC flag set in order to be lockable.
+    if (unlikely(type == D3DRTYPE_VOLUMETEXTURE && desc.Pool == D3DPOOL_DEFAULT && !(desc.Usage & D3DUSAGE_DYNAMIC)))
```

Comment:
Nice catch... except, ugh, we don't. Because at least the Wine tests expect ATI1/2 (both FOURCCs) to fail here. Guess I'll have to test this more extensively, but I'll modify IsLockable directly.

Reply:
@K0bin So, turns out fourccs aren't exempt for volume textures. Why? Because d3d9 says so. Adjusted IsLockable to reflect this.